docs/src/content/docs/examples/hn.mdx

```diff
@@ -1,14 +1,87 @@
 ---
-title: Hacker News sentiment analysis
-description: Analyzing Hacker News sentiment with GenSX
+title: Hacker News Analyzer
+description: Extract out key trends from top Hacker News posts
 sidebar:
   order: 1
 ---
 
-GenSX is a simple framework for building complex generative AI and LLM workflows.
+The [Hacker News Analyzer](https://github.com/cortexclick/gensx/tree/main/examples/hackerNewsAnalyzer) example uses GenSX to fetch, analyze, and extract trends from the top Hacker News posts. It shows how to combine data fetching, parallel analysis, and content generation in a single workflow to create two outputs: a detailed report and a tweet of the trends.
 
-It uses `jsx` to define and orchestrate workflows with functional, reusable components:
+## Workflow
+
+The Hacker News Analyzer workflow is composed of the following steps:
+
+1. Fetch the top 500 Hacker News posts (`<HnCollector>`)
```

Comment:
Might be worth noting that this is limited to `text` posts. We skip analyzing any link posts that would require web scraping for convenience of the example.